clickstream/src/main/kotlin/clickstream/internal/DefaultClickStream.kt

```diff
@@ -44,6 +45,14 @@ internal class DefaultClickStream private constructor(
         }
     }
 
+    override fun addEventListener(listener: CSEventListener) {
```

Comment:
this is prohibited if we allow this, the future if we need similar mechanism it would bloated the class

Reply:
Plus one. Also, I believe the cognitive load to follow through this chain-of-command cannot be overlooked.